package.json

```diff
@@ -112,7 +117,6 @@
     "lodash-es": "npm:lodash@^4.17.11",
     "loglevel": "^1.7.0",
     "matrix-js-sdk": "^9.0.1",
-    "metamask-extension-provider": "^1.1.1",
```

Comment:
Why re-inventing `metamask-extension-provider`?

Reply:
current one does not support `provider.request()`, see #1860

Reply:
shouldn't we PR upstream instead of re-invent the wheel?

Reply:
There's a long time to wait... I see you PR before https://github.com/MetaMask/extension-provider/pull/8

Reply:
Otherwise, we can fork a temporal package for MetaMask/extension-provider. Before they accept our PR we can use the forked version.

Reply:
I agree